baselines/FedStar/pyproject.toml

```diff
@@ -0,0 +1,135 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "FedStar" # <----- Ensure it matches the name of your baseline directory containing all the source code
```

Comment:
```suggestion
name = "fedstar" # <----- Ensure it matches the name of your baseline directory containing all the source code
```

Reply:
Following my comment above, you need to re-name the directory using a lowercase word (e.g. `fedstar` instead of `FedStar`). This is needed for Poetry to install the `fedstar` package properly apparently.

Reply:
Thanks for the suggestion, I have updated the folder name and project name in pyproject toml file